themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -381,7 +381,7 @@ root-url.tooltip=Root URL appended to relative URLs
 valid-redirect-uris=Valid Redirect URIs
 valid-redirect-uris.tooltip=Valid URI pattern a browser can redirect to after a successful login or logout. Simple wildcards are allowed such as 'http://example.com/*'. Relative path can be specified too such as /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.
 valid-post-logout-redirect-uris=Valid post logout redirect URIs
-valid-post-logout-redirect-uris.tooltip=Valid URI pattern a browser can redirect to after a successful logout. A value of '+' will use the list of valid redirect uris. Simple wildcards are allowed such as 'http://example.com/*'. Relative path can be specified too such as /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.
+valid-post-logout-redirect-uris.tooltip=Valid URI pattern a browser can redirect to after a successful logout. A value of '+' will use the list of valid redirect uris. A value of '-' will use an empty list of valid redirect uris. Simple wildcards are allowed such as 'http://example.com/*'. Relative path can be specified too such as /my/relative/path/*. Relative paths are relative to the client root URL, or if none is specified the auth server root URL is used. For SAML, you must set valid URI patterns if you are relying on the consumer service URL embedded with the login request.
```

Comment:
Maybe use "A value of '+' or empty field will..." and "A value of '-' will not allow any post logout redirect URIs".?

Reply:
> Maybe use "A value of '+' or empty field will..." and "A value of '-' will not allow any post logout redirect URIs".?

I like that; I'll update the PR.

Reply:
+1 for this point.

When you are at this, it may be also good to also remove last sentence of the tolltip (The sentece starting with `For SAML, ...`) because Post Logout Redirect URIs are not applicable for SAML clients.

Reply:
Thanks for removing the SAML part. Do you plan to change it also in a way as proposed like `A value of '+' or empty field will...` . There is nothing about empty field in the tooltip and IMO, it will be good to add it.

Reply:
Done